core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.ts

```diff
@@ -395,6 +395,7 @@ angular.module('oppia').directive('audioTranslationBar', [
               AudioPlayerService.load(audioTranslation.filename)
                 .then(function() {
                   $scope.audioLoadingIndicatorIsShown = false;
+                  $scope.isLoadingAudio = false;
```

Comment:
Maybe rename to `$scope.AudioIsLoading` (see style [guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript)).

Reply:
ok

Reply:
done